### ISSUE TITLE
Several Bugs with PRO-C

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuuhakuzx/procfw#455]
```
All the files say PRO-B instead of PRO-C.
The cIPL Installer for 6.60 says FW ERROR! <- Same mistake like with PRO-B10!
```

Original issue reported on code.google.com by `mirzet.w...@gmx.de` on 2 Jun 2012 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
Can't test this as I have no CIPL hackable unit...
Did you use the Nightly or the Release Build?

Nightlies are known to produce bad CIPL installers due to a Compiler Bug...
```

Original comment by `gentlema...@gmail.com` on 2 Jun 2012 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
As i said, tried the PRO-C1 build for 6.60!
```

Original comment by `mirzet.w...@gmx.de` on 2 Jun 2012 at 4:35

**Comment by GoogleCodeExporter** [comment]
```
There you go, 6.60 PRO-C (non fix) with not working cIPL flasher (download via 
official google code page)
And i did not used the cIPL flasher of the 6.39 archive, i used the one of the 
6.60 archive. But this bug got fixed since 6.60 PRO-Cfix1/fix2 ;)

http://www.youtube.com/watch?v=NbX9QeDXqZQ
```

Original comment by `mirzet.w...@gmx.de` on 2 Jun 2012 at 5:47

**Comment by GoogleCodeExporter** [comment]
```
Was fixed in 
https://code.google.com/p/procfw/source/detail?r=4c4517dbc366715d58825f496b3a3d7
3221aed45
```

Original comment by `devnonam...@gmail.com` on 4 May 2014 at 12:33
- Changed state: **Fixed**
